.hound.yml

```diff
@@ -1,4 +1,3 @@
-# Prefer single quotes
-StringLiterals:
-  EnforcedStyle: single_quotes
-  Enabled: true
```

Comment:
Can I use this changes in [`feature_migrating_virtual_machine_templates`](https://github.com/dice-cyfronet/atmosphere/tree/feature_migrating_virtual_machine_templates)?